toolchain/check/testdata/class/extend_adapt.carbon

```diff
@@ -94,8 +94,7 @@ class StructAdapter {
 // CHECK:STDOUT:   %.1: type = unbound_element_type SomeClass, i32 [template]
 // CHECK:STDOUT:   %.2: type = struct_type {.a: i32, .b: i32} [template]
 // CHECK:STDOUT:   %.3: type = ptr_type {.a: i32, .b: i32} [template]
-// CHECK:STDOUT:   %.4: type = ptr_type SomeClass [template]
```

Comment:
Is this expected churn?

Reply:
Yes -- the value representation for the adapter is now copied from the adapted type. It used to be `SomeClass*`, which was why this type showed up, but it's now `{.a: i32, .b: i32}*`.